### ISSUE TITLE
0.14.4: error: use of undeclared identifier 'RTAX_MAX', etc.

### ISSUE THREAD

**Issue by yurivict** [paullouisageneau/libdatachannel#483]
```
/disk-samsung/freebsd-ports/www/libdatachannel/work/libdatachannel-0.14.4/deps/usrsctp/usrsctplib/user_recv_thread.c:90:18: error: use of undeclared identifier 'RTAX_MAX'
        for (i = 0; i < RTAX_MAX; i++) {
                        ^
/disk-samsung/freebsd-ports/www/libdatachannel/work/libdatachannel-0.14.4/deps/usrsctp/usrsctplib/user_recv_thread.c:107:15: error: use of undeclared identifier 'RTM_NEWADDR'
        if ((type != RTM_NEWADDR) && (type != RTM_DELADDR)) {
                     ^
/disk-samsung/freebsd-ports/www/libdatachannel/work/libdatachannel-0.14.4/deps/usrsctp/usrsctplib/user_recv_thread.c:107:40: error: use of undeclared identifier 'RTM_DELADDR'
        if ((type != RTM_NEWADDR) && (type != RTM_DELADDR)) {
                                              ^
/disk-samsung/freebsd-ports/www/libdatachannel/work/libdatachannel-0.14.4/deps/usrsctp/usrsctplib/user_recv_thread.c:126:14: error: use of undeclared identifier 'RTM_NEWADDR'
        if (type == RTM_NEWADDR) {
                    ^
/disk-samsung/freebsd-ports/www/libdatachannel/work/libdatachannel-0.14.4/deps/usrsctp/usrsctplib/user_recv_thread.c:150:33: error: use of undeclared identifier 'RTAX_MAX'
        struct sockaddr *sa, *rti_info[RTAX_MAX];
                                       ^
/disk-samsung/freebsd-ports/www/libdatachannel/work/libdatachannel-0.14.4/deps/usrsctp/usrsctplib/user_recv_thread.c:160:26: error: use of undeclared identifier 'RTM_DELADDR'
                        if (ifa->ifam_type != RTM_DELADDR && ifa->ifam_type != RTM_NEWADDR) {
                                              ^
/disk-samsung/freebsd-ports/www/libdatachannel/work/libdatachannel-0.14.4/deps/usrsctp/usrsctplib/user_recv_thread.c:160:59: error: use of undeclared identifier 'RTM_NEWADDR'
                        if (ifa->ifam_type != RTM_DELADDR && ifa->ifam_type != RTM_NEWADDR) {
                                                                               ^
/disk-samsung/freebsd-ports/www/libdatachannel/work/libdatachannel-0.14.4/deps/usrsctp/usrsctplib/user_recv_thread.c:164:42: error: use of undeclared identifier 'rti_info'
                        sctp_get_rtaddrs(ifa->ifam_addrs, sa, rti_info);
                                                              ^
/disk-samsung/freebsd-ports/www/libdatachannel/work/libdatachannel-0.14.4/deps/usrsctp/usrsctplib/user_recv_thread.c:166:9: error: use of undeclared identifier 'RTM_DELADDR'
                        case RTM_DELADDR:
                             ^
/disk-samsung/freebsd-ports/www/libdatachannel/work/libdatachannel-0.14.4/deps/usrsctp/usrsctplib/user_recv_thread.c:167:9: error: use of undeclared identifier 'RTM_NEWADDR'
                        case RTM_NEWADDR:
                             ^
/disk-samsung/freebsd-ports/www/libdatachannel/work/libdatachannel-0.14.4/deps/usrsctp/usrsctplib/user_recv_thread.c:168:57: error: use of undeclared identifier 'rti_info'
                                sctp_handle_ifamsg(ifa->ifam_type, ifa->ifam_index, rti_info[RTAX_IFA]);
                                                                                    ^
/disk-samsung/freebsd-ports/www/libdatachannel/work/libdatachannel-0.14.4/deps/usrsctp/usrsctplib/user_recv_thread.c:168:66: error: use of undeclared identifier 'RTAX_IFA'
                                sctp_handle_ifamsg(ifa->ifam_type, ifa->ifam_index, rti_info[RTAX_IFA]);
                                                                                             ^
12 errors generated.
```

clang-11
FreeBSD 13

**Comment by paullouisageneau** [comment]
This is an issue with [usrsctp](https://github.com/sctplab/usrsctp/). It might actually be solved in the usrsctp submodule version shipped with 0.15.0.

**Comment by paullouisageneau** [comment]
This should be fixed by 44e3667caa88d10bf548efa2a9ce2968369edd96, please re-open if needed.
